package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ably",
   "description": "Realtime client library for Ably, the realtime messaging service",
-  "version": "2.4.1",
+  "version": "2.5.0",
```

Comment:
this was probably missed when committing, we should not change package version here

Reply:
Corrected, thanks!